### PR TITLE
Updated rustup to 1.24.3 and fixed readme file format issues that vio…

### PR DESCRIPTION
**Pull request by mason-chase**
…lated Markdown standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,20 +43,20 @@ $ make debuild derebuild
...
+environment and installs build dependencies. The primary advantage of using debian
 packaging tools in this case is to leverage the [Reproducible
 Builds](https://wiki.debian.org/ReproducibleBuilds) project. While building a debian
 package, `debuild` will record the names and versions of all detected build dependencies
 into a *.buildinfo file. The Reproducible Builds Project's `derebuild.pl` script can then
 read the buildinfo file to drill down in the [Debian Snapshot
 Archive](http://snapshot.debian.org/) to output the list of packages and generate an apt
-sources.list which should contain all of those packages. The list of packages should then
-be checked in as build-deps in the enclave/docker/ folder, along with sources.list and
+sources. list which should contain all of those packages. The list of packages should then
```
Comment (kalvdans) [inline README.md:53]:
```suggestion
`sources.list` which should contain all of those packages. The list of packages should then
```

**Comment by mason-chase** [comment]
Hi Everyone,

I'm facing an error when I try to build the docker:

any idea how to fix this?

```
cds_enclave 0.1.0] cargo:rustc-link-search=native=/home/rust/src/build/target/release/build/cds_enclave-a361ebe00517ac0f/out
error: Cannot set `RUSTC_BOOTSTRAP=1` from build script of `cds_enclave v0.1.0 (/home/rust/src/cds_enclave)`.
note: Crates cannot set `RUSTC_BOOTSTRAP` themselves, as doing so would subvert the stability guarantees of Rust for your project.
help: If you're sure you want to do this in your project, set the environment variable `RUSTC_BOOTSTRAP=cds_enclave` before running cargo instead.
warning: build failed, waiting for other jobs to finish...
error: build failed
make[2]: *** [Makefile:269: build/target/release/libcds_enclave.a] Error 101
```

Cheers

**Comment by mason-chase** [comment]
anyone?

**Comment by mason-chase** [comment]
still waiting for someone to review

**Comment by mullick1998** [comment]
hello,
while building the enclave using docker for [ContactDiscoveryService](https://github.com/signalapp/ContactDiscoveryService), I'm getting this error. 
```
mv ../*.buildinfo debian/buildinfo
mv ../*.deb debian/
make: *** No rule to make target 'build/libkbupd_enclave.hardened.signed.so', needed by 'sign'.  Stop.
make: Leaving directory '/home/rust/src'
```

I'm using SGX enabled device, and  fulfilled all the prerequisites.

So I tried the option without docker and Debian, While building first I fulfilled all the prerequisites by installing the **sgx sdk** and **sgxpsw**. Then while building first it gave me an error i.e. ```error[E0554]: `#![feature]` may not be used on the stable release channel```

I installed **rustup** and **nightly toolchain** and it gave me an another error regarding error: ```Cannot set `RUSTC_BOOTSTRAP=1` from build script of `cds_enclave v0.1.0 (~/signal/ContactDiscoveryService/enclave/cds_enclave)`. note: Crates cannot set `RUSTC_BOOTSTRAP` themselves, as doing so would subvert the stability guarantees of Rust for your project. help: If you're sure you want to do this in your project, set the environment variable `RUSTC_BOOTSTRAP=cds_enclave` before running cargo instead.```

So as the note, I added ```RUSTC_BOOTSTRAP=cds_enclave```, and after that it gave me an error which seems a dead-end i.e ```ModuleNotFoundError: No module named 'pyxed' ``` I checked the enclave build folder it has pyxed but it is missing **xed-interface.h** in it., ie ```includes.h:11:10: fatal error: xed-interface.h: No such file or directory```

I even tried to get the [Pyxed](https://github.com/huku-/pyxed) seperately, and compile it but it shows some error which isn't fixed yet.

What could be the possible solution to this?

**Pull request closed without merging**